general-rules.md

```diff
@@ -20,4 +20,4 @@
 
 8. Genrally, you have to follow the instructions of moderators and staff members.
 
-9. It is important to note that this server is not a public forum. Final say in everything is up to the moderators and staff team. The moderators and staff team of the server reserve the right to punish for any reason as long as there is a consensus among said moderators and staff members.
\ No newline at end of file
+9. It is important to note that this server is not a public forum. Everything is up to the moderators and staff team. The moderators and staff team of the server reserve the right to punish for any reason as long as there is a consensus among said moderators and staff members.
```

Comment:
Please note that this server does not intend to imitate a public forum. Moderators and staff members have final say in the enforcement of these rules or other infractions and punishment given for reasons not listed is subjectable to staff's discretion. Don't attempt to loophole or use semantics to get around these rules. Respect the intent behind them instead of the literal text.